src/main.rs

```diff
@@ -1,5 +1,22 @@
-use equilibrium::app_main;
+mod fluid;
+use crate::fluid::{Fluid, FluidConfig, SimulationConfig};
 
 fn main() {
-    app_main();
+    let fluid_config = FluidConfig {
+        ..Default::default()
+    };
```

Comment:
I think this can be written easier with FluidConfig::default() directly

Reply:
Done